dali/operators/reader/loader/coco_loader.cc

```diff
@@ -199,8 +204,9 @@ void parse_annotations(
         // That means that the mask encoding is not polygons but RLE (iscrowd==1),
         // which is not needed for instance segmentation
         if (parser.PeekType() != kArrayType) {
-          while (parser.NextObjectKey()) {}
-          break;
+          to_add = false;
+          parser.SkipValue();
+          continue;
```

Comment:
Why not:
```
to_add = false;
parser.SkipObject();
break;
```
Maybe we should add some method to Annotation class that would check if it is valid/complete, and check if it has bboxes/segm_meta_/segm_coords_ and image_id_, category_id_ set.

Reply:
You're right, `SkipObject` is the way to go

> Maybe we should add some method to Annotation class that would check if it is valid/complete, and check if it has bboxes/segm_meta_/segm_coords_ and image_id_, category_id_ set.

Doing this instead of the `to_add` logic?